kubert/src/admin.rs

```diff
@@ -29,18 +31,18 @@ pub struct AdminArgs {
 
 /// Supports configuring an admin server
 #[cfg_attr(docsrs, doc(cfg(feature = "admin")))]
-#[derive(Clone, Debug)]
 pub struct Builder {
```

Comment:
Losing `Clone` for `Builder` is unfortunate, but probably not a hard blocker. At the very least we should manually implement `Debug` impls where we've lost them.